storage/service_account/hmac/list_keys.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hmac
+
+// [START storage_list_hmac_keys]
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+// List all HMAC keys associated with the project.
+func listHMACKeys(w io.Writer, projectID string) ([]*storage.HMACKey, error) {
+
+	ctx := context.Background()
+
+	// Initialize client.
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+		return nil, err
+	}
+	defer client.Close() // Closing the client safely cleans up background resources.
+
+	iter := client.ListHMACKeys(ctx, projectID)
+	var gotKeys []*storage.HMACKey
+	for {
+		key, err := iter.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			// Handle error.
+			return nil, err
+		}
+		fmt.Fprintf(w, "The HMAC key metadata is:\n")
```

Comment:
This is inconsistent with other languages - 
we only printed out ServiceAccountEmail and AccessId of the keys

Reply:
Ah, I made this mistake in the Python samples as well-- will submit a PR to fix up that as well.

Reply:
Also incorrect for Ruby, which is what led me astray I think. See https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/storage/hmac.rb#L15 . cc: @frankyn

Reply:
Fixed in python and looks like @frankyn has a fix up for Ruby as well.

Reply:
+1, and PHP. They're pending review. Thanks for the ping @tritone!